client/src/redux/index.js

```diff
@@ -95,6 +98,9 @@ export const appMount = createAction(types.appMount);
 export const tryToShowDonationModal = createAction(
   types.tryToShowDonationModal
 );
+
+export const reportGaEvent = createAction(types.reportGaEvent);
```

Comment:
@raisedadead Here is the action. just removed calling the react.ga events from the components and relocated them to a SAGA.

Reply:
please clarify if you had something else in mind.

Reply:
What would be these calls: https://github.com/freeCodeCamp/freeCodeCamp/pull/37984/files#diff-e5939f8d99bc04b87018776785947561R74

Reply:
@raisedadead, only ga.events were included in the action. 
Now I included all GA is called from there and should be ready for full review.